Bridge/Bridge.sol

```diff
@@ -381,13 +393,23 @@ contract Bridge is EthTokenReciever {
                 acceptedEthTokens[tokenAddress],
                 "The Token is not accepted for transfer to sidechain"
             );
-            uint256 balanceBefore = token.balanceOf(address(this));
-            token.safeTransferFrom(msg.sender, address(this), amount);
-            uint256 balanceAfter = token.balanceOf(address(this));
-            require(
-                balanceAfter - balanceBefore >= amount,
-                "Not enough tokens transferred"
-            );
+            if (tokenAddress == address(_addressUSDT)) {
+                uint256 balanceBefore = _addressUSDT.balanceOf(address(this));
```

Comment:
balanceOf in usdt and other ERC20 are the same, so i guess there is no need to split using if